doc/LICENSE.txt

```diff
@@ -8,7 +8,7 @@ Copyright (c) 2012-2014, Arne Schwabe
 
 The name ics-openvpn and OpenVPN for Android refer to the same program.
 OpenVPN for Android is distributed under the GPL license version 2 (see Below)
-with the following clarification/additional terms of what constitues "deriviate work":
+with the following clarification/additional terms of what constitutes "derivative work":
```

Comment:
Is this ok?

Reply:
yeah that one is okay. That my own mistake but changing other people's license text is something that I am not comfortable with.